src/setIntersection.js

```diff
@@ -0,0 +1,12 @@
+export default function setIntersection(arr1, arr2) {
+  
+  var resultsArray = []
```

Comment:
Avoid appending 'array' at the end of your variable identifiers. Call it 'results' to increase readability, and so you won't have to change each identifier if you decide to use another data structure to hold your results later on down the line. This is good practice for learning how to write maintainable code.